adafruit_rfm69.py

```diff
@@ -474,8 +474,10 @@ def operation_mode(self, val):
         op_mode |= val << 2
         self._write_u8(_REG_OP_MODE, op_mode)
         # Wait for mode to change by polling interrupt bit.
+        start = time.monotonic()
         while not self.mode_ready:
-            pass
+            if (time.monotonic() - start) >= 1:
+                raise RuntimeError("Operation mode failed to set.")
```

Comment:
> I normally write code for my own consumption, so I would like a little feedback regarding RuntimeError being the correct error type. Also whether the message is clear enough/. I'm already thinking that "Operation mode changed timed out." might be more descriptive.

Instead of that, change `RuntimeError` to a `TimeoutError: Operate mode failed to set.`. Using a `TimeoutError` is more self-descriptive of what happened (mode was not set in a specific time) than a `RuntimeError`

Reply:
Thanks. Great suggestion. Change made.

Reply:
Thank you.